suites/os/suite.js

```diff
@@ -66,6 +66,8 @@ module.exports = {
 		});
 
 		this.suite.teardown.register(() => {
+			this.log('Removing image');
+			fse.unlinkSync('/data/image');
```

Comment:
A comment here about what exactly we deleting would be nice for your future selves.

Reply:
And, now for balenaOS folks as well.